kernel/packages/entryPoints/website.ts

```diff
@@ -91,15 +93,20 @@ namespace webApp {
 
   export async function loadUnity({ instancedJS }: InitializeUnityResult) {
     const i = (await instancedJS).unityInterface
+    let worldConfig: WorldConfig = globalThis.globalStore.getState().meta.config.world!
+
+    if (ENABLE_OLD_SETTINGS) {
+      worldConfig.enableOldSettings = ENABLE_OLD_SETTINGS
+    }
 
     i.ConfigureHUDElement(HUDElementID.MINIMAP, { active: true, visible: true })
     i.ConfigureHUDElement(HUDElementID.NOTIFICATION, { active: true, visible: true })
     i.ConfigureHUDElement(HUDElementID.AVATAR_EDITOR, {
       active: true,
       visible: OPEN_AVATAR_EDITOR
     })
-    i.ConfigureHUDElement(HUDElementID.SETTINGS, { active: ENABLE_OLD_SETTINGS, visible: false })
-    i.ConfigureHUDElement(HUDElementID.SETTINGS_PANEL, { active: !ENABLE_OLD_SETTINGS, visible: false })
+    i.ConfigureHUDElement(HUDElementID.SETTINGS, { active: worldConfig.enableOldSettings!, visible: false })
+    i.ConfigureHUDElement(HUDElementID.SETTINGS_PANEL, { active: !worldConfig.enableOldSettings, visible: false })
```

Comment:
this seems odd... cause it mixing some undefined check with boolean negation.
can we have here something like `const useOldChat : boolean` and use that instead?

Reply:
Done!